### ISSUE TITLE
can't work with c header file

### ISSUE THREAD

**Issue by fcying** [Badhi/nvim-treesitter-cpp-tools#38]
When I use `TSCppDefineClassFunc` in c header file, the following error occurs. After removing the macro, it works fine.

```
E5108: Error executing lua ...reesitter-cpp-tools/lua/nt-cpp-tools/output_handlers.lua:13: attempt to perform arithmetic on field 'class_end_row' (a nil value)
stack traceback:
...reesitter-cpp-tools/lua/nt-cpp-tools/output_handlers.lua:13: in function <...reesitter-cpp-tools/lua/nt-cpp-tools/output_handlers.lua:8>
.../nvim-treesitter-cpp-tools/lua/nt-cpp-tools/internal.lua:321: in function 'run'
[string ":lua"]:1: in main chunk
```

test file
```
#ifndef _T_H
#define _T_H

#include <stdint.h>

uint8_t test(void);

#endif


``

**Comment by 7AXEL** [comment]
Check out this project: https://github.com/7AXEL/Web2APK

**Comment by Badhi** [comment]
@fcying, Thanks for filing bug. I will fix it as soon as possible

**Comment by Badhi** [comment]
FIxed with #39. Please give it a try

**Comment by fcying** [comment]
Thanks, it is working properly now
